ansible_collections/arista/avd/roles/eos_validate_state/defaults/main.yml

```diff
@@ -21,3 +21,6 @@ output_dir: '{{ root_dir }}/{{ output_dir_name }}'
 # Output for structured YAML files:
 structured_dir_name: 'structured_configs'
 structured_dir: '{{ output_dir }}/{{ structured_dir_name }}'
+
+# Allow different manufacturers
+accepted_xcvr_manufacturers: "{{ validation_role.xcvr_own_manufacturers + ['Not Present', 'Arastra, Inc.', 'Arista Networks'] }}"
```

Comment:
missing documentation update on this new knob.
Also I think it is unnecessary complex to take one knob as input and have hidden builtin extras. Would be simpler to just set the right knob with the default value and leave it to the user to set `accepted_xcvr_manufacturers` directly and include the defaults when they add more. Maybe they want to _not_ allow one of the defaults.